magda-gateway/src/Authenticator.ts

```diff
@@ -5,14 +5,27 @@ import * as passport from "passport";
 import * as URI from "urijs";
 import * as signature from "cookie-signature";
 
+/** This is present in the express-session types but not actually exported properly, so it needs to be copy-pasted here */
+type SessionCookieOptions = {
+    maxAge?: number;
+    signed?: boolean;
+    expires?: Date;
+    httpOnly?: boolean;
+    path?: string;
+    domain?: string;
+    secure?: boolean | "auto";
+    encode?: (val: string) => string;
+    sameSite?: boolean | "lax" | "strict" | "none";
+};
+
 export interface AuthenticatorOptions {
     sessionSecret: string;
     dbPool: pg.Pool;
-    cookieOptions?: express.CookieOptions;
+    cookieOptions?: SessionCookieOptions;
 }
 
 export const DEFAULT_SESSION_COOKIE_NAME: string = "connect.sid";
-export let DEFAULT_SESSION_COOKIE_OPTIONS: express.CookieOptions = {
+export let DEFAULT_SESSION_COOKIE_OPTIONS = {
```

Comment:
Should here be `DEFAULT_SESSION_COOKIE_OPTIONS : SessionCookieOptions` so that if we decide to change the default session cookie options, it's still covered by type checking?